libparsec/crates/protocol/tests/vlob.rs

```diff
@@ -179,18 +179,28 @@ fn serde_vlob_create_req(#[case] raw: &[u8], #[case] expected: authenticated_cmd
         strictly_greater_than: "2000-1-2T01:00:00Z".parse().unwrap(),
     }
 )]
+#[should_panic(expected = "missing field `backend_timestamp`")]
 #[case::bad_timestamp_legacy(
-    // Generated from Python implementation (Parsec v2.8.1+dev)
+    // Generated from Python implementation (Parsec v2.11.1+dev)
```

Comment:
Why the version have changed here ?

Reply:
Because copy-paste is not always a friend! I'll fix that ^^

Reply:
Done